rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -300,6 +304,10 @@ def sample(
                 index_triple[1],
             )
             episode = self.episodes[episode_idx]
+
+            # If we use random n-step sampling, draw the n-step for this item.
+            if _random_n_step:
+                n_step = int(self.rng.integers(_n_steps[0], _n_steps[1]))
```

Comment:
Call this var something else. We should not mix/override function args with local vars.

E.g.:
```
if random_n_step:
    actual_n_step = int(self.rng.integers(n_step[0], n_step[1]))
```
^ here, we know (b/c random_n_step is True), that the given arg `n_step` must be a tuple, so we can use it directly here w/o having to go the detour via the `_n_steps` var (which we don't need in this func).

Sorry, lots of nits, but this makes the code much more readable.